build.sbt

```diff
@@ -75,3 +75,12 @@ addCommandAlias(
     " -rff current.throughput.prof.csv" +
     " -prof stack;"
 )
+
+val generateMasterFileTask =
+  taskKey[Unit]("If the master file has not been created, current is copied. This will happen just the first time.")
+generateMasterFileTask := {
+  if (!file("bench/master.avgtime.csv").exists())
+    IO.copy(Seq((file("bench/current.avgtime.csv"), file("bench/master.avgtime.csv"))))
+}
+
+addCommandAlias("generateMasterFile", ";" + generateMasterFileTask.key)
```

Comment:
**question**: Why do you need this command alias?

Reply:
Actually it's not needed. I thought it was, but I was wrong.